src/Intellenum.SharedTypes/IntellenumValidationException.cs

```diff
@@ -17,15 +16,8 @@ public IntellenumException(string message) : base(message)
     public IntellenumException(string message, Exception inner) : base(message, inner)
     {
     }
-
-    protected IntellenumException(
```

Comment:
These have been obsoleted in .NET 8, so I've removed them, but maybe they should be included in older versions?

Reply:
I agree - removing them is the best thing. I'd also like to only support the most recent frameworks. This isn't a massive problem as far as features are concerned, but it does complicate things somewhat. Also, supporting many frameworks mean that there's tens of thousands of snapshot comparison files to verify the generated output.

Reply:
Yeah that's fair. I think it would make sense to get rid of .NET Core 3.1 and .NET 5 as they've been out of support for a while now. .NET 7 is technically out of support as well but it's only been a few weeks.